packages/signature/src/cryptography.ts

```diff
@@ -49,6 +49,9 @@ export class Signature implements Hashable, Serializable {
     public readonly buffer: Buffer
   ) {
     if (buffer.length !== 65) { throw TypeError() }
+    if ([27, 28].includes(this.recovery)) {
+      throw new Error('invalid signature. should be V is 27 or 28.')
```

Comment:
Please fix to
'invalid signature. V should be 27 or 28.'
or maybe you can just say
'invalid signature'